src/commands/deploy.yml

```
@@ -5,65 +5,64 @@ parameters:
     type: string
     description: The path where your AWS SAM template file is located.
     default: ""
-  stack-name:
+  stack_name:
     type: string
     description: The name of the AWS CloudFormation stack you're deploying to. If you specify an existing stack, the command updates the stack. If you specify a new stack, the command creates it.
   capabilities:
     type: string
     description: Comma separated list of capabilities for AWS Cloudformation to create stacks. The only valid values are CAPABILITY_IAM and CAPABILITY_NAMED_IAM. If you have IAM resources, you can specify either capability. If you have IAM resources with custom names, you must specify CAPABILITY_NAMED_IAM.
     default: "CAPABILITY_IAM"
-  profile-name:
+  profile_name:
     type: string
     description: Select a specific profile from your credential file to get AWS credentials.
     default: "default"
-  aws-region:
+  region:
     description: |
-      Env var of AWS region to operate in
       (defaults to AWS_DEFAULT_REGION)
-    type: env_var_name
-    default: AWS_DEFAULT_REGION
+    type: string
+    default: ${AWS_DEFAULT_REGION}
   debug:
     type: boolean
     description: Turns on debug logging.
     default: false
-  parameter-overrides:
+  parameter_overrides:
     type: string
     description: AWS CloudFormation parameter overrides encoded as key=value pairs. Use the same format as the AWS CLI.
     default: ""
-  no-fail-on-empty-changeset:
+  no_fail_on_empty_changeset:
     type: boolean
     description: Specify if deploy command should return a zero exit code if there are no changes to be made to the stack.
     default: true
-  image-repositories:
+  image_repositories:
     type: string
     description: Comma separated list of the ECR image repositories to push the Lambda container. If this option is set, the SAM package command will be run automatically.
     default: ""
-  s3-bucket:
+  s3_bucket:
     type: string
     description: The name of the S3 bucket where this command uploads the artifacts that are referenced in your template. If this option is set, the SAM package command will be run automatically.
     default: ""
-  resolve-s3:
+  resolve_s3:
     type: boolean
-    description: Automatically create an Amazon S3 bucket to use for packaging and deploying for non-guided deployments. If you specify the --guided option, then the AWS SAM CLI ignores --resolve-s3. If you specify both the --s3-bucket and --resolve-s3 options, then an error occurs.
+    description: Automatically create an Amazon S3 bucket to use for packaging and deploying for non-guided deployments. If you specify the --guided option, then the AWS SAM CLI ignores --resolve_s3. If you specify both the --s3_bucket and --resolve_s3 options, then an error occurs.
```

If we are referring to the CLI's parameters then it should be:

```suggestion
    description: Automatically create an Amazon S3 bucket to use for packaging and deploying for non-guided deployments. If you specify the --guided option, then the AWS SAM CLI ignores --resolve-s3. If you specify both the --s3-bucket and --resolve-s3 options, then an error occurs.
```